coalib/bearlib/abstractions/Linter.py

```diff
@@ -695,6 +740,11 @@ def linter(executable: str,
         A custom message that gets displayed when ``check_prerequisites``
         fails while invoking ``prerequisite_check_command``. Can only be
         provided together with ``prerequisite_check_command``.
+    :param global_bear:
+        Wether the created bear should be a global bear or not. Global bears
```

Comment:
Wether -> Whether

and maybe I would also do
global bear -> GlobalBear (enclosed in double backticks)